doc/manuals/user/known_limitations.md

```diff
@@ -29,29 +26,20 @@ Or, if you are sending a huge request, this one:
 carefully" text. Developers of the HTTP library in which Orion Context
 Broker is based seem to be funny guys :) :)
 
-If this 1 MB limit doesn't work for you, you can change it using the CLI option -payloadMaxSize.
-But, this is untested territory. At your own risk ...
+If this 1 MB limit doesn't work for you, you can change it using the [CLI option](../admin/cli.md) `-inReqPayloadMaxSize`.
+However, before doing this please have a look to [performance considerations](../admin/perf_tuning.md#payload-and-message-size-and-performance).
 
 ## Notification maximum size
 
-The maximum size of notification payloads defaults to 8MB. Larger notifications will not be sent by the context broker and you
-will get the following trace in the log file:
+Notification maximum size (including HTTP request line, headers and payload) is set to 8MB by default.
+Larger notifications will not be sent by the context broker and you will get the following trace in the log file:
 
     HTTP request to send is too large: N bytes
 
 where N is the number of bytes of the too large notification.
 
-However, you can change this limit by starting the broker with the CLI option `-notifMaxSize <size in bytes>`
-
-Note also that there is another message size limit, one that is valid for **all** outgoing messages from the broker:
-
-* Normal responses
-* Notifications
-* Forwarded messages
-
-This last message size limit is about the size of the entire message, not only the payload and its default value is 8MB, which can
-be modified using the CLI option `-outMsgMaxSize`.
-
+However, you can change this limit by starting the broker with the [CLI option](../admin/cli.md) `-outReqMsgMaxSize`.
+However, before doing this please have a look to [performance considerations](../admin/perf_tuning.md#payload-and-message-size-and-performance).
```

Comment:
Before doing this please have a look *at*

That second 'However' doesn't sound good.

Reply:
Reworded to use only one "however". Fixed in e38ae720da8468a4cd2a35029c4f8c6a07dfa9c6